app/views/modules/_header.html.slim

```diff
@@ -1,3 +1,9 @@
 - if current_user
-  div.header
-    = button_to "Log out", logout_path, class: "btn btn-default pull-right"
+  .container-fluid
+    nav.navbar.navbar-default.navbar-fixed-top
+      a.navbar-brand href="http://www.wediko.org" 
```

Comment:
I'd prefer if we instead had the logo go back to the root page and a smaller link go back to Wediko's main site. A big link that leaves the application altogether isn't great UX.